lib/modules/onc_program_procedure.yml

```diff
@@ -0,0 +1,1219 @@
+procedure:
```

Comment:
JB: When writing a program to make this into a matrix, or however you planned on publishing a non-developer friendly version of this, it would be helpful to populate the matrix with the inferno_tests reference ID and the inferno_tests name (and potentially even description). Additionally, it might be helpful to parse the specific part of the test procedure language that is met by the Inferno test. In top of this, there seems to be tests that are listed several times across different test procedure steps; it would be interesting to see a matrix of all the inferno tests and the corresponding lists of test procedure steps they support.